### PR TITLE
Convert lines to compound operators

### PR DESCRIPTION
**Pull request by idontyboi**
## Changes

Converts various lines in various files to represent them with the newer Luau compound operators. 

Tutorials and partially Luau directories are left alone, since I am not sure about them.

## Checks

By submitting your pull request for review, you agree to the following:

- [X] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [X] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [X] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

**Comment by Relukn** [comment]
Do you want regex patterns to the other operators? Otherwise, I can make an additional pr.

**Comment by idontyboi** [comment]
Used Notepad++ regex to find duplicate words, and looked through each result one by one: `\b([\w\.]+) \= \g1`. Not sure if it caught every single thing. Left out concat operators.

Should concat operators or the two left-out directories get involved in this?

**Comment by Relukn** [comment]
The only concat operator I think should be changed is in the Luau and C# Comparison (/creator-docs/content/en-us/luau/luau-csharp-comparison.md). That article already uses `+=`, and people reading it probably know what `..=` would mean.

I think it's fine to update the Luau directory, but the tutorials would probably be better as is.

**Comment by idontyboi** [comment]
Updated the luau-csharp-comparison.md, metatables.md files since they are more advanced topics. Left control-structures.md and scope.md alone.

**Review by Relukn — APPROVED** [review]

File: content/en-us/luau/luau-csharp-comparison.md [hunk 1]
```diff
@@ -361,7 +361,7 @@ In Luau, you can write variables and logic in a tighter scope than their functio
 local example = "Example text"
 
 do
-	local example = example .. " changed!"
+	example ..= " changed!"
```
Comment (aetter) [inline content/en-us/luau/luau-csharp-comparison.md:364]: This is cool -- didn't realize it was an option.

**Review by aetter — APPROVED** [review]

**Comment by aetter** [comment]
Thanks @idontyboi and @Relukn!

**Pull request merged**